packages/site/src/features/cronjob-snap/Cronjob.tsx

```diff
@@ -0,0 +1,60 @@
+import { FunctionComponent } from 'react';
+import { Button } from 'react-bootstrap';
+import { Result, Snap } from '../../components';
+import { useInvokeMutation } from '../../api';
+import { getSnapId } from '../../utils/id';
+
+const CRONJOB_SNAP_ID = 'npm:@metamask/test-snap-cronjob';
+const CRONJOB_SNAP_PORT = 8010;
+
+export const Cronjob: FunctionComponent = () => {
+  const [invokeSnap, { isLoading, data, error }] = useInvokeMutation();
+
+  const handleFireCronjob = () => {
```

Comment:
I don't think we need these functions and buttons to trigger the cronjob dialog?

Reply:
fixed